src/NuGet.Core/NuGet.ProjectModel/ProjectStyle.cs

```diff
@@ -16,7 +16,7 @@ public enum ProjectStyle : ushort
         ProjectJson = 1,
 
         /// <summary>
-        /// MSBuild style, project.assets.json is generated in the RestoreOutputPath folder
+        /// MSBuild style, project.assets.json is generated in the RestoreOutputPath (aka MSBuildProjectExtensionsPath) folder
```

Comment:
If you are changing it, it'd be better if we just say MSBuildProjectExtensionsPath.

Reply:
Response: https://github.com/NuGet/NuGet.Client/pull/3270#discussion_r465657566